fever.yaml

```diff
@@ -83,6 +83,19 @@ context:
   submission-url: amqp://guest:guest@localhost:5672/
   submission-exchange: context
 
+# Configuration for Stenosis (persisted full packet capture).
+stenosis:
+  enable: false
+  submission-url: http://localhost:19205/
+  submission-timeout: 5s
+  # Set to true and parameters below to enable TLS connection to Stenosis
+  tls: false
```

Comment:
More a suggestion: add a tls-section:

```
  tls:
    enabled: false
    client-chain-...
```

This makes it easier to configure and grouping options is 'safer'.

Reply:
I can see the appeal, but I'd rather not change existing configuration following the existing schema.